dlme_airflow/tasks/harvest_validator.py

```diff
@@ -0,0 +1,30 @@
+import pandas as pd
+
+from airflow import DAG
+from airflow.operators.python import BranchPythonOperator
+from airflow.utils.task_group import TaskGroup
+
+from dlme_airflow.utils.dataframe import dataframe_from_s3
+
+
+def validate_harvest(task_instance, task, **kwargs):
+    collection = kwargs["collection"]
+    current_harvest = pd.read_csv(
+        task_instance.xcom_pull(task_ids=task.upstream_task_ids)[0]
+    )
+    previous_harvest = dataframe_from_s3(collection)
+
+    if current_harvest.equals(previous_harvest):
+        return f"{collection.provider.name.upper()}_ETL.{collection.name}_etl.skip_load_data"
```

Comment:
Have you looked to see if this works with our Collection only DAGS in `auc.yaml`?

Reply:
That's a good observation, I don't think it will as I see those labels are lowercased... I'll fix that up. Thanks.

Reply:
Let me know if you need help, I had to puzzle through this myself.

Reply:
![Screen Shot 2022-09-30 at 9 01 29 AM](https://user-images.githubusercontent.com/2294288/193310769-4271afda-7ad0-46e9-bcde-e05b070f7155.png)

I think this will be OK, at least it draws properly. I'll need to run this image in AWS to verify. I'll finish that up today so this can be merged later.

Reply:
I do think it will be worth adding a test for it though, so I'll do that this morning as well.